DataFormats/Headers/include/Headers/DataHeader.h

```diff
@@ -711,10 +713,20 @@ struct DataHeader : public BaseHeader {
   //___NEW STUFF GOES BELOW
 
   ///
-  /// first orbit of time frame as unique identifier within the run
+  /// first orbit of time frame as unique identifier within the run (unless the TF is replayed), since v2
```

Comment:
>(unless the TF is replayed)

is this still relevant?

Reply:
Replaying? I think so, why not?

Reply:
I thought we need to modify the DataProcessing header to utilize all pipelines? Is there another reason to modify these fields wehen repeating TFs?

Reply:
sorry, I am afraid I don't understand what you mean. What has changed in this: (1) we need the 1st orbit of the TF consistent with the obits in TF payload for reconstruction; (2) we need a unique identifier of the TF; (3) the 1st orbit cannot be as a uniqueID in case of replay, which I believe, we also need ?

Reply:
I'm talking the case when replying with looping of the same TFs. Your comment says "(unless the TF is replayed)". 
Because (1), we cannot modify this field when looping TFs, which makes the comment misleading. 
The TF uniqueness in the looping mode (2), for the sake of DPL-pipeline routing, will be hadled now by the DataProcessingHeader::startTime? or also by chainging the DataHeader::tfCounter to never wrap?
(3) is a conesquence of (1), for that reason the comment IMO should be just: `/// first orbit of time frame`

Reply:
ok, I did not realize you are commenting on the comment, will change this. To me the mTFCounter can substitute the 
DataProcessingHeader::startTime.  As for the wrapping of the mTFCounter, as a counter starting from 0 and incremented for every TF, it would take running for 497 days of continued running with 10ms TFs to wrap it.